app/_src/kubernetes-ingress-controller/concepts/architecture.md

```diff
@@ -14,7 +14,17 @@ The {{site.kic_product_name}} configures {{site.base_gateway}} using Ingress or
 
 The figure illustrates how {{site.kic_product_name}} works:
 
-![high-level-design](/assets/images/products/kubernetes-ingress-controller/high-level-design.png "High Level Design")
+<!--vale off-->
+{% mermaid %}
+flowchart TD
+    subgraph Kubernetes cluster
+        direction LR
+        A(<img src="/assets/images/icons/third-party/kubernetes-logo.png" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> API server) --> |events| B(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Controller)
+        B --> |configuration| C(<img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/>Kong)
+        C --> D(services)
```

Comment:
Looks like this is missing the green arrow that services traffic:

https://docs.konghq.com/kubernetes-ingress-controller/latest/concepts/architecture/

The red arrows are events and config from Kubernetes, and the green arrows are traffic through the proxy.

I'd add another object outside of the "Kubernetes cluster" subgraph that's called something like "request traffic" ?